charts/invenio/templates/_helpers.tpl

```diff
@@ -1,3 +1,53 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "invenio.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "invenio.fullname" -}}
```

Comment:
Seems like you define `invenio.fullname` but never use it.

Reply:
Removed.